css/style.css

```diff
@@ -0,0 +1,199 @@
+.main{
+    display: flex;
+    flex-direction: column;
+    max-width: 1280px;
+    margin: 0 auto;
+    padding: 0 20px;
+    position: relative;
+}
+
+#header{
+    display: flex;
+    max-width: 100%;
+    flex-direction: row;
+    align-items: center;
+    padding: 15px, 20px;
+    position: fixed;
+    left: 5%;
+    right: 5%;
+    top: 0;
+   
+}
+
+.logo{
+    display: flex;
+    
+}
+
+.logo img{
+    width: 100%;
+    border-radius: 50%;
+    width: 70px;
+}
+
+#nav-bar{
+    display: flex;
+    flex-grow: 1;
+    min-height: 80px;
+    align-items: center;
+    display: inline;
+}
+
+
+
+#nav-bar ul{
+
+    text-align: center;
+    list-style-type: none;
+    display: flex;
+    width: 100%;
+    gap: 50px;
+    margin-left: 800px;
+    
```

Comment:
Check spaces and follow CSS best practices here: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Organizing